storage/remote/codec.go

```diff
@@ -538,3 +541,13 @@ func labelsToLabelsProto(labels labels.Labels, buf []prompb.Label) []prompb.Labe
 	}
 	return result
 }
+
+// Used for hashing configs and diff'ing hashes in ApplyConfig.
+func toHash(data interface{}) (string, error) {
```

Comment:
`codec.go` seems like a weird place for this function, perhaps it should just go next to `ApplyConfig` in `storage.go`?

Reply:
Just put it there because we're using it in multiple places, `storage.go`, `write.go`, `write_test.go`, and `client_test.go`. Not a big deal to me where the function goes though.